libraries/AP_InertialSensor/AP_InertialSensor.h

```diff
@@ -423,6 +441,13 @@ class AP_InertialSensor : AP_AccelCal_Client
     LowPassFilter2pVector3f _gyro_filter[INS_MAX_INSTANCES];
     Vector3f _accel_filtered[INS_MAX_INSTANCES];
     Vector3f _gyro_filtered[INS_MAX_INSTANCES];
+    // Thread-safe public version of _last_raw_gyro
```

Comment:
should these be ifdef based on DSP enable?

Reply:
Fixed